opaque_keys/__init__.py

```diff
@@ -20,6 +18,9 @@
 )
 from stevedore.enabled import EnabledExtensionManager
 
+# pylint: disable=wrong-import-order
```

Comment:
Question: why was it necessary to break the expected import order?

Reply:
it fails on quality check for `python3` and if the suggested changes are made those fail on `python2`

> third party import "from six import iteritems, python_2_unicode_compatible, text_type, viewkeys, viewitems, with_metaclass" should be placed before "from _collections import defaultdict" (wrong-import-order)
third party import "from stevedore.enabled import EnabledExtensionManager" should be placed before "from _collections import defaultdict" (wrong-import-order)